### ISSUE TITLE
[Accessibility]: No change on clicking on match in ref panel

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36583]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33510

### Problem description

When clicking on a matching line in the ref panel, the file content side pane (code view?) opens up, but the focus is still on the button. There is no screen reader announcement that anything happened.

### Expected behavior

Arguably the focus should shift to the code view and there should be a screen reader announcement indicating what happened.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
